index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Did this need to change when it's Spanish?

Reply:
It probably should but I haven't noticed it causing any issues in my testing or in production. I'm not sure that it makes a difference if it's switched dynamically on the client.